wcomponents-theme/unbuilt.package.json

```diff
@@ -18,6 +18,8 @@
 		"handlebars": "~4.0.5",
 		"i18next": "~3.4.1",
 		"i18next-xhr-backend": "~1.0.1",
```

Comment:
I was thinking along similar but more extreme lines: make the two packages into a property defined in the POM. @ricksbrown thoughts on this?